Face_Toolbox_Overview.adoc

```diff
@@ -141,6 +141,33 @@ The tools and media for the creation of artefacts are defined for all tests in t
 .. The evaluator shall wear the 3D face mask and present it to the TOE or present the 3D mannequin head to the TOE under the identical controlled environment used during enrolment.
 .. The evaluator shall present the 3D face mask or mannequin head in a way that minimizes reflections from ambient lighting.
 
+== Stored Artefact Usage
+If stored artefacts will be used for testing, a number of fresh artefacts are still required to ensure that stored artefacts have not degraded in non-visible ways that would effect the testing. Stored artefacts also have maximum ages based on the date of initial creation (weeks are used for consistent time periods). Stored artefacts can only be used up to the maximum age based on the time difference between the creation date and the use date (use must be completed before the maximum age allowed).
+
+The following table specifies the maximum amount of stored artefacts that can be used for each type and the maximum age allowed for the artefacts to be used.
+
+[cols=".^3,.^2",options="header",]
```

Comment:
Table has only two columns.
This line should be changed to 
[cols=".^2,.^3,.^3",options="header",]
Please do the same change to the other toolboxes